tests/execution/auth-basic-file.md

```diff
@@ -0,0 +1,78 @@
+# Auth with BasicAuth loaded from const env
+
+```graphql @server
+schema
+  @server(port: 8000, graphiql: true, auth: [{id: "basic", basic: {htpasswd: "{{link.htpasswd}}"}}])
```

Comment:
```suggestion
  @server(port: 8000, graphiql: true)
```

Drop auth setting from server. We can implicitly understand that linking to an htpasswd file means the config requires authentication.